cirq-superstaq/docs/getting_started_cirq_superstaq.ipynb

```diff
@@ -186,14 +186,6 @@
     "print(\"Job status: \", job.status())\n",
     "print(job.counts())"
    ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "845f5925",
-   "metadata": {},
-   "outputs": [],
-   "source": []
```

Comment:
Maybe a nit, but does this need to be deleted?

Reply:
It was a blank cell at the end which I assumed didn't serve a specific purpose. The other notebooks don't have empty cells like that, so I deleted them to be more standard.